.gitignore

```diff
@@ -35,3 +35,6 @@ jspm_packages
 
 # Optional REPL history
 .node_repl_history
+
+lib
```

Comment:
I'm confused about the goal. i thought the goal was to make this easier to use in a js version agnostic way. i'm unclear about the benefit of having users run the module's build task.

or maybe i'm conflating git check in with npm check in.